web-ui/src/index.js

```diff
@@ -5,12 +5,7 @@ import App from "./App";
 import * as serviceWorker from "./serviceWorker";
 import "font-awesome/css/font-awesome.min.css";
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById("root")
-);
+ReactDOM.render(<App />, document.getElementById("root"));
```

Comment:
Curious as to what lead to this change....

Reply:
I remember having a error specifically related to this so I removed it, but I can't seem to replicate it anymore so I just added it back in.

Reply:
Found it, here is the error if left in StrictMode
index.js:1 Warning: findDOMNode is deprecated in StrictMode. findDOMNode was passed an instance of Transition which is inside StrictMode.

Reply:
do you know what is utilizing that api?